CMakeLists.txt

```diff
@@ -221,6 +219,10 @@ if(APHRODITE_GPU_LANG STREQUAL "CUDA")
     "kernels/all_reduce/custom_all_reduce.cu")
 
   # Add CUTLASS and GPTQ Marlin kernels if not MSVC
```

Comment:
This comment might be inappropriate here

Reply:
Missed it, it's fixed now